release.py

```diff
@@ -256,11 +260,105 @@ def command_stage_docs():
 
     sys.exit(0)
 
+def command_release_announcement_email():
+    release_version_num = raw_input('What is the current release version:')
+    previos_release_version_num = raw_input('What is the previous release version:')
+    number_of_contributors = int(subprocess.check_output('git shortlog -sn --no-merges %s..%s | wc -l' % (previos_release_version_num, release_version_num) , shell=True))
+    contributors = subprocess.check_output("git shortlog -sn --no-merges %s..%s | cut -f2 | tr '\n' ',' | sed -e 's/,/, /g'" % (previos_release_version_num, release_version_num), shell=True)
```

Comment:
This command has always left a trailing `,`. You could potentially omit the commands after the `cut` and just do a split/join in python that will give exactly what we want.

Also, not sure if it was intentional or not, but this command seems to elide the alphabetical sorting that's in the command on the wiki.

Reply:
this is done